src/main/java/org/sigmah/shared/dto/element/ContactListElementDTO.java

```diff
@@ -263,6 +699,19 @@ public void setMember(boolean isMember) {
     set(IS_MEMBER, isMember);
   }
 
+  public CheckboxElementDTO getCheckboxElement() {
+		return get(CHECKBOX_ELEMENT);
+	}
+
+  public Integer getCheckboxElementId() {
+    return getCheckboxElement() == null ? null : getCheckboxElement().getId();
+  }
+
+  public void setCheckboxElement(CheckboxElementDTO checkboxElement) {
+		set(CHECKBOX_ELEMENT, checkboxElement);
+	}
+
```

Comment:
Please, fix indentation of `getCheckboxElement` and `setCheckboxElement`: (hint: @xavier-calland should ckeck his IDEA configuration 😉 )